templates/_layout.html

```diff
@@ -21,6 +21,7 @@
 
     <link rel="stylesheet" href="{{ static_url('css/styles.css') }}" />
 
+    <script src="https://assets.ubuntu.com/v1/842d27d3-lazysizes.min.js" async></script>
```

Comment:
It probably doesn't need to be the very first one script? Could we move it later, so browser can pick more important ones first? Probably could go below {% block scripts_includes %}